### DISCUSSION TITLE
Allow environment variables for volume `subpath` and `transformSubpath`

### DISCUSSION THREAD

**Discussion by thomasvantuycom** [craftcms/cms#15087] [Ideas] [answered]
Most filesystem and volume settings allow values to be configured using environment variables. However, the `subpath` and `transformSubpath` settings for volumes are notable exceptions. Could this behavior be modified? I'm willing to submit a pull request if there is interest.

**Comment by brandonkelly** [comment, marked as answer]
Easy enough! Just added environment variable support to those two settings for Craft 5.2 (e52a28259ea68a80508ea27aa4153ddb9bb17a93).

**Reply by brandonkelly** [reply]
5.2.0 is out with that change.
